src/main/java/de/tum/in/www1/artemis/domain/Achievement.java

```diff
@@ -51,12 +57,15 @@
     public Achievement() {
     }
 
-    public Achievement(String title, String description, String icon, AchievementRank rank, AchievementType type, Course course, Exercise exercise) {
+    public Achievement(String title, String description, String icon, AchievementRank rank, AchievementType type, Long successCriteria, Long minScoreToQualify, Course course,
+            Exercise exercise) {
```

Comment:
I think all instances of `Achievement` created in this PR set the `exercise` parameter to `null`. Can we remove this parameter if it is really unused? (Or is that just planning for the future?)

Reply:
This was a suggestion of @krusche but so far this parameter is actually unused.